packages/config/config/devices/0x010f/fgms001.json

```diff
@@ -761,7 +803,7 @@
 			"#": "86",
 			"$if": "firmwareVersion >= 3.2",
 			"label": "LED - temperature for blue colour",
-			"description": "Minimum temperature resulting in blue LED illumination",
+			"description": "When the temperature is below this (°C), the light is blue.",
```

Comment:
The unit should be specified using the `"unit"` property, not as part of the description.